### PR TITLE
ZALLY-73 Show Help if Parameters are Missing

### LINKED ISSUE #73: Help instructions should show up if arguments were not sufficient in any form
At the moment, CLI only shows help instructions if '--help' parameter is supplied. I think it's a better user experience if these help instructions always show up when there is an issue with supplied argument list - except if we can give better error description.

### PR DESCRIPTION
**Pull request by netme**
Closes #73 

📗

### CODE REVIEW

**Comment by netme** [comment]
@maxim-tschumak could you please also post a comment with 👍 ? Zappr doesn't understand GH approvals yet...

**Comment by maxim-tschumak** [comment]
:+1:

File: cli/src/main/java/de/zalando/zally/cli/Main.java [hunk 1]
```diff
@@ -45,7 +47,9 @@ void run(String[] args) {
 
     private boolean lint(String[] args) throws RuntimeException {
         if (args.length < 1) {
-            throw new RuntimeException("Please provide a swagger file path or URL");
+            System.err.println("Please provide a swagger file path or URL\n");
+            parser.showHelp();
+            throw new RuntimeException("");
```
Comment (fmueller) [inline cli/src/main/java/de/zalando/zally/cli/Main.java]: If you don't have a message for the exception you can also use empty constructor parameter list: `new RuntimeException()`

Comment (netme) [inline cli/src/main/java/de/zalando/zally/cli/Main.java]: @fmueller, true, I've already tried it, but the library will print `(null)` to the console in this case. That's why I've decided to use this hack :-)

Comment (fmueller) [inline cli/src/main/java/de/zalando/zally/cli/Main.java]: Ah ok, makes sense :)

**Pull request merged**